apps/docs/pages/guides/auth/auth-helpers/nextjs.mdx

```diff
@@ -171,7 +171,8 @@ export async function GET(request) {
   const code = requestUrl.searchParams.get('code')
 
   if (code) {
-    const supabase = createRouteHandlerClient({ cookies })
+    const cookieStore = () => cookies()
+    const supabase = createRouteHandlerClient({ cookies: () => cookieStore })
```

Comment:
Nice. Confirming this should be a double callback?

ie:
```ts
createRouteHandlerClient({ cookies: () => () => cookies() })
```

Reply:
Good catch, that is indeed incorrect. This is what find and replace does after replacing one before running it.

Reply:
Oh wait, they are all wrong.